views/layout.html

```diff
@@ -26,6 +26,7 @@
   <!--[if !IE 8]><!-->
   <link rel="stylesheet" type="text/css" media="all" href="//{{CDN_HOST}}/stylesheets/govuk-frontend/v4.6.0/govuk-frontend-4.6.0.min.css" />
   <link rel="stylesheet" type="text/css" media="all" href="//{{CDN_HOST}}/stylesheets/services/overseas-entities/application.css"  />
+  {% include "includes/styles.html" %}
```

Comment:
Can you remove this change for now so that we can get this passed sonar cube then we will put this in a separate PR